### PR TITLE
add link-to-redmine feature

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
this is totally untested, but I think it should do what we wanted?

**Review by ekohl — COMMENTED** [review]
I think we'll need to add the permissions to add comments to the application. I can try it out on the test instance.

File: prprocessor/__main__.py [hunk 1]
```diff
@@ -293,7 +304,7 @@ async def run_pull_request_check(pull_request: Mapping, check_run=None) -> bool:
         },
     )
 
-    return conclusion == 'success'
+    return (conclusion == 'success', issue_results, invalid_commits)
```
Comment (evgeni) [inline prprocessor/__main__.py:307]: no consumer uses `invalid_commits`, so we could also drop it from the return, but I thought maybe it's useful at some point later

**Comment by evgeni** [comment]
I was the only one complaining this is missing and I got used to using the "checks" tab, so closing for now.

**Pull request closed without merging**